INSTALL.md

```diff
@@ -39,6 +40,7 @@ cd mquery
 vim .env
 docker compose -f docker-compose.dev.yml up  # this will take a while
 docker compose exec dev-web python3 -m mquery.db
+docker compose exec dev-web python3 -m alembic upgrade head
```

Comment:
don't change the documentation. After #421 alembic will be run with web automatically anyway.